includes/class-wc-stripe-helper.php

```diff
@@ -443,6 +443,25 @@ public static function get_order_by_setup_intent_id( $intent_id ) {
 		return false;
 	}
 
+	/**
+	 * Sanizite and retrieve a shortened statement descriptor.
+	 *
+	 * The descriptor will be used as prefix and the order ID will be concatenated, acting as a suffix.
+	 *
+	 * @param string $statement_descriptor
+	 * @param WC_Order $order
+	 * @return string $statement_descriptor Final shortened statement descriptor
+	 */
+	public static function get_shortened_statement_descriptor( $statement_descriptor = '', $order = null ) {
+		$statement_descriptor = self::clean_statement_descriptor( $statement_descriptor );
+
+		if ( method_exists( $order, 'get_id' ) && ! empty( $order->get_id() ) ) {
+			$statement_descriptor = $statement_descriptor . '* #' . $order->get_id();
```

Comment:
Do I understand correctly that we're doing this concatenation (joined by `* `) ourselves instead of passing the suffix (as described in [this doc section](https://stripe.com/docs/statement-descriptors#dynamic)) because that is the only way for us to manage the shortened descriptor (prefix) outside of the Stripe Dashboard?

Reply:
Yes, that's correct. The thing is if we pass the suffix, it will be merged with the shortened descriptor from the Stripe account and not from the value set in the settings. Even if we try to pass both `statement_descriptor` and the `statement_descriptor_suffix` won't work, the "prefix" is not customizable at all.